### ISSUE TITLE
Package fails to build with custom SRCEXT configuraion

### ISSUE THREAD

**Issue by ikubaku** [eagletmt/guzuta#5]
Omakase build fails with the following error if SRCEXT(makepkg.conf) is other than '.src.tar.gz' (such as '.src.tar').

```
thread 'main' panicked at 'Unable to add PKGBUILDs/yay to abs tarball foo/os/x86_64/foo.abs.tar.gz', /home/ikubaku/.cargo/registry/src/github.com-1ecc6299db9ec823/guzuta-0.3.0/src/main.rs:588:17
```

The error appears to be triggered here https://github.com/eagletmt/guzuta/blob/4b64608510a7f6960ea55720ad729303d45920c7/src/abs.rs#L67 because the source package is generated by the local makepkg and is an uncompressed archive.

**Comment by eagletmt** [comment]
Right... it supports .src.tar.gz only. I'm going to remove all features related to abs.

Background: The reason guzuta requires source packages is to create an abs tarball.
The abs tarball was used by [abs](https://git.archlinux.org/abs.git) and [yaourt](https://github.com/archlinuxfr/yaourt) but all projects are dead now.

**Comment by eagletmt** [comment]
Released v0.4.0 including #6 changes
